fxa-oauth-server/lib/routes/destroy.js

```diff
@@ -16,8 +16,9 @@ module.exports = {
     payload: Joi.object().keys({
       client_secret: Joi.string().allow(''),
       access_token: validators.token,
-      refresh_token: validators.token
-    }).rename('token', 'access_token').xor('access_token', 'refresh_token')
+      refresh_token: validators.token,
+      refresh_token_id: validators.token,
```

Comment:
When is a `refresh_token_id` passed instead of just a `refresh_token`? And why?

Reply:
Explained here: https://github.com/mozilla/fxa-auth-server/pull/2937#discussion_r266286874